pytorch_lightning/utilities/cli.py

```diff
@@ -228,6 +294,17 @@ def add_arguments_to_parser(self, parser: LightningArgumentParser) -> None:
             parser: The argument parser object to which arguments can be added
         """
 
+    def link_optimizers_and_lr_schedulers(self) -> None:
+        """Creates argument links for optimizers and lr_schedulers that specified a link_to"""
+        for key in self.parser.optimizers_and_lr_schedulers.keys():
+            class_type, link_to = self.parser.optimizers_and_lr_schedulers[key]
+            if link_to != 'AUTOMATIC':
```

Comment:
could use negation + "continue" to save an indentation level :)

Reply:
Done